admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+  - Use the services provided by the API resource **`/settlementModels`**.
+- View transaction details for a particular transfer.
+  - Use the services provided by the API resource **`/transactions`**.
+
+### Supported API services
+
+[Table "Supported API services"](#table-supported-api-services) includes high-level descriptions of the services that the API provides. For more detailed information, see the sections that follow.
+
+| URI  |  HTTP method `GET`| HTTP method `PUT` | HTTP method `POST` | HTTP method `DELETE` |
+|---|---|---|---|---|
+| **`/participants`**  | Get information about all participants  | Not supported  | Create participants in the Hub  |  Not supported |
+| `/participants/limits`  | View limits for all participants  |  Not supported | Not supported  | Not supported  |
+| `/participants/{name}`  | Get information about a particular participant  | Update participant details (activate/deactivate a participant)  | Not supported  |  Not supported |
+| `/participants/{name}/endpoints`  | View participant endpoints  | Not supported  | Add/Update participant endpoints  | Not supported  |
+| `/participants/{name}/limits`  | View participant limits  |  Adjust participant limits | Not supported  |  Not supported |
+| `/participants/{name}/positions`  | View participant positions  | Not supported  | Not supported  | Not supported  |
+| `/participants/{name}/accounts`  | View participant accounts and balances  |  Not supported | Create Hub accounts  |  Not supported |
+| `/participants/{name}/accounts/{id}`  |  Not supported | Update participant accounts  | Record Funds In or Out of participant account  | Not supported  |
+| `/participants/{name}/accounts/{id}/transfers/{transferId}`  | Not supported  | Not supported  | Record a Transfer as a Funds In or Out transaction for a participant account  |  Not supported |
+| `/participants/{name}/initialPositionAndLimits`  | Not supported  | Not supported  |  Add initial participant limits and position | Not supported  |
+| **`/settlementModels`**  | View all settlement models  | Not supported  | Create a settlement model  |  Not supported |
+| `/settlementModels/{name}`  | View settlement model by name  | Update a settlement model (activate/deactivate a settlement model) | Not supported  |  Not supported |
+| **`/transactions/{id}`**  | Retrieve transaction details by `transferId`  | Not supported  | Not supported  | Not supported |
+
+## API Resource `/participants`
+
+The services provided by the resource `/participants` are primarily used by the Hub Operator for viewing, creating, and updating participant-related details, such as limit (Net Debit Cap), position, or endpoints configured.
+
+### `GET /participants`
+
+The `GET /participants` request is used to retrieve information about all participants.
+
+#### Data model
+
+The `GET /participants` request sends an empty request body.

```

Comment:
This section would be infinitely more readable with examples of requests and responses for each endpoint.

Stripe has good examples of this - see https://stripe.com/docs/api/balance/balance_retrieve

Reply:
Agree, will add those in. My brief said I should follow the conventions employed by the FSPIOP API doc, but I agree, examples definitely add a lot of value.

Reply:
thanks :)